internal/test_cases/utils.go

```
@@ -48,8 +47,14 @@ func logReadableFileContents(logger *logger.Logger, fileContents string) {
 	printableFileContents = regex1.ReplaceAllString(printableFileContents, "<|SPACE|>")
 	printableFileContents = regex2.ReplaceAllString(printableFileContents, "<|SPACE|>")
 
+	// This is of the form "test-N"
+	oldPrefix := logger.GetSecondaryPrefix()
+	testNumber := strings.TrimPrefix(oldPrefix, "test-")
+	logger.UpdateSecondaryPrefix(fmt.Sprintf("test-%s.lox", testNumber))
+	defer logger.UpdateSecondaryPrefix(oldPrefix)
+
 	for _, line := range strings.Split(printableFileContents, "\n") {
-		logger.Plainf(color.YellowString("[test.lox]") + " " + line)
+		logger.Plainf(line)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to (*github.com/codecrafters-io/tester-utils/logger.Logger).Plainf

(govet)